src/functions/forecast/list-allocations/mockData6.json

```diff
@@ -0,0 +1,6 @@
+{
+    "headers": {
+      "Content-Type": "application/json"
+    },
```

Comment:
I think you should not try to merge mockData to develop

Reply:
done